### ISSUE TITLE
Last Update date for migrated themes should not be the date of the migration

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-server#8558]
STR:

1. Load AMO -dev
2. Open the details page of a migrated theme - https://addons-dev.allizom.org/en-US/firefox/addon/building-a-snowman/
3. Observe the Last Update date value

Actual result:
The Last Update date is the date of the migration

Expected result:
The Last Update date is the date when the theme author made the last changes

Notes:
 - reproduced on AMO -dev with FF60, Win10x64

![image](https://user-images.githubusercontent.com/31961530/41539361-4beaa38a-7316-11e8-956f-9178ddfe16a5.png)

**Comment by eviljeff** [comment]
yeah, I'm going to wait until after #8405 is fixed (and maybe #8412)

**Comment by AlexandraMoga** [comment]
I've verified with a static theme migrated yesterday and it seems that the Last Update date is still the date of the migration:

https://addons-dev.allizom.org/en-US/firefox/addon/litefox/?src=search

![image](https://user-images.githubusercontent.com/31961530/42020782-9091e43c-7ac1-11e8-89aa-9bf7deaff64e.png)

**Comment by ioanarusiczki** [comment]
@eviljeff I checked the dates for the sample above https://addons-dev.allizom.org/en-US/firefox/addon/litefox/?src=search and also for
https://addons-dev.allizom.org/en-US/firefox/addon/an-autumn-reverie/?src=search
https://addons-dev.allizom.org/en-US/firefox/addon/lilac-prediction/?src=search
and the date is still June 27,2018 which is the date of migration as shown in the comment above.

**Comment by eviljeff** [comment]
The fix didn't update any themes migrated previously (it would have been technically possible, but it's only -dev so i didn't bother).  But newly migrated themes should retain the dates from the LWT.

https://addons-dev.allizom.org/en-US/firefox/addon/tigers-matter/ was migrated yesterday to test updates.  It looks good right now - if it's still got the old 'last update' date after today's stats run (can't remember when that is) then I think this is fixed.

**Comment by ioanarusiczki** [comment]
@eviljeff This https://addons-dev.allizom.org/en-US/firefox/addon/tigers-matter/ looks good to me too.
I also checked for another sample https://addons-dev.allizom.org/en-US/firefox/addon/grass_2/ that was recently migrated. 
Marking this verified fixed on AMO dev with FF61(Win10). 
![date for a migrated lw theme](https://user-images.githubusercontent.com/33448286/42564643-fc66deb8-8509-11e8-86ca-ab9d26e569b5.png)
